cmd/kp/main.go

```diff
@@ -148,7 +151,7 @@ func getClusterBuilderCommand(clientSetProvider k8s.ClientSetProvider) *cobra.Co
 	clusterBuilderRootCmd := &cobra.Command{
 		Use:     "clusterbuilder",
 		Short:   "ClusterBuilder Commands",
-		Aliases: []string{"cb"},
+		Aliases: []string{"clusterbuilders", "cb"},
```

Comment:
if cb is an option, should we also have cbs to keep the plurals uniform?

Reply:
same for clusterstores and clusterstacks

Reply:
Interesting thought... curious to know what others think about it.

Reply:
why not!

Reply:
Could we add the other k8s shortnames? (clstrbldr,clstrbldrs) https://github.com/pivotal/kpack/blob/8407183ed0ca7690683ddfdea2f5bab3473e8215/config/clusterbuilder.yaml#L13-L14

Reply:
Also, we should probably add cb/cbs to the kubernetes short name list?

Reply:
> Also, we should probably add cb/cbs to the kubernetes short name list?

I do not know what is behaviour if short names collide, when registering a custom resource in kubernetes. looking around for the impact